cron/hourly/70-cancel-subscriptions-for-cancelled-orders.ts

```diff
@@ -28,24 +29,38 @@ const getHostFromOrder = async order => {
   return models.Collective.findByPk(transactions[0].HostCollectiveId);
 };
 
+const getOrderCancelationReason = (collective, order, orderHost) => {
+  if (!order.Tier) {
+    return `Order tier deleted`;
+  } else if (collective.deactivatedAt) {
+    return `@${collective.slug} archived their account`;
+  } else if (!collective.HostCollectiveId) {
+    return `@${collective.slug} was un-hosted`;
+  } else if (collective.HostCollectiveId !== orderHost.id) {
+    return `@${collective.slug} changed host`;
+  } else {
+    return `Order cancelled`;
+  }
```

Comment:
This `getOrderCancelationReason` function is really cool, I'm glad that we're able to extract the reason with such a simple code.